pdns/dnsdistdist/docs/reference/dq.rst

```diff
@@ -205,6 +205,14 @@ This state can be modified from the various hooks.
 
     :returns: A :ref:`timespec` object
 
+  .. method:: DNSQuestion::getElapsedUs -> double
```

Comment:
We try to sort the methods alphabetically (you know the difference between theory and practice, right?), so would you mind moving this entry just below `getEDNSOptions`?